science/multimon-ng/files/patch-cmakelists.txt_pulseaudio.diff

```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 73c8422..085e7b3 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -31,8 +31,8 @@ elseif( UNIX )
+ 	endif( X11_FOUND )
+ 	find_package( PulseAudio )
+ 	if ( PULSEAUDIO_FOUND )
+-		option( PULSE_AUDIO_SUPPORT "Enable pulse audio support" ${PULSEAUDIO_FOUND} )
```

Comment:
Is there no way to enable/disable pulse audio support via cmake options ? It would be better to avoid a patch file if possible.

Reply:
Unfortunately not, otherwise why I used the patch? And I haven't the time to send a patch upstream.

Reply:
Just looking at the cmake file, have you tried passing the option -DPULSE_AUDIO_SUPPORT=0 ?

Reply:
Ops, sorry for the arrogance!  How I couldn't see that! Thank you!

Reply:
Thanks, thats better 😉